src/style.css

```
@@ -1 +1,65 @@
-/* Styles go here */
+html {
+  scroll-behavior: smooth;
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  font-size: 12px;
+  text-transform: uppercase;
+  font-weight: 500;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  position: sticky;
+  top: 0;
+  min-width: 900px;
+  padding: 10px 50px;
```

add only horizontal paddings
then add height to header or list or link
```suggestion
  padding: 0 50px;
```